src/font/lv_font.c

```diff
@@ -46,16 +47,46 @@ const void * lv_font_get_glyph_bitmap(lv_font_glyph_dsc_t * g_dsc, lv_draw_buf_t
 {
     const lv_font_t * font_p = g_dsc->resolved_font;
     LV_ASSERT_NULL(font_p);
+#if LV_FONT_CACHE_GLYPH_CNT > 0
+    /* If the font cache is enabled, try to use it to get the glyph bitmap */
+    return lv_font_cache_get_glyph_bitmap(g_dsc, draw_buf);
+#else
     return font_p->get_glyph_bitmap(g_dsc, draw_buf);
+#endif
+}
+
+lv_result_t lv_font_get_glyph_raw_bitmap(lv_font_glyph_dsc_t * g_dsc, lv_draw_buf_t * draw_buf)
```

Comment:
Sorry just noticed the inconsitency (although you implemented it as I suggested):

```suggestion
lv_result_t lv_font_get_glyph_static_bitmap(lv_font_glyph_dsc_t * g_dsc, lv_draw_buf_t * draw_buf)
```
Let's use the `static_bitmap` expression everywhere.

Reply:
Done.